src/main/webapp/app/modules/canvas/visualization/canvas.tsx

```diff
@@ -132,6 +133,10 @@ const Canvas = (props: VisualizationProp) => {
     }
   }, []);
 
+  useEffect(() => {
+    setShowLoader(props.visualLoader);
+  }, [props.visualLoader]);
+
```

Comment:
no need of this code...props.visualLoader is sufficient and rename this to isLoaderOn